src/modules/News/routes.ts

```diff
@@ -24,4 +24,16 @@ router.get(
   newsController.getTopHeadlines,
 );
 
+router.get(
+  '/sources',
+  validateRequest(newsValidationSchemas.sources),
+  newsController.getSources,
+);
+
+router.get(
+  '/search',
```

Comment:
The authMiddleware is missing in the /search. We should add it according to the requirements:
![image](https://github.com/user-attachments/assets/da599f21-5654-4113-93e2-98daa2b4ef95)

Reply:
good catch thank you! I will add this

Reply:
Hey @regikov @Francieli-S I was wondering why is the auth needed for search. That would mean a user should be auth in order to search news. Also according to the list from the initial requirements the route for top headlines and personalized should also be auth. But, does it make sense to have auth routes so a user have to be logged in, in order to see the news.
This will also impact the ui, what can be seen with or without auth.
Same for the personalized, they are pesonalized on a category not if a user has logged in. @ahzia

Reply:
I think for reading any news a user doesn't have to neccesairy make a profile and be logged in. But for features like Bookmarks and Comments, yes.
@regikov @Francieli-S @ahzia